### ISSUE TITLE
Not found: fietsersbond.nl

### ISSUE THREAD

**Issue by Eddict** [navossoc/KeePass-Yet-Another-Favicon-Downloader#64]
https://www.fietsersbond.nl
MS Edge shows the icon

page source:
```
<link rel="apple-touch-icon" sizes="180x180" href="https://www.fietsersbond.nl/wp-content/themes/tabula/apple-touch-icon.png">
<link rel="icon" type="image/png" href="https://www.fietsersbond.nl/wp-content/themes/tabula/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="https://www.fietsersbond.nl/wp-content/themes/tabula/favicon-16x16.png" sizes="16x16">
<link rel="manifest" href="https://www.fietsersbond.nl/wp-content/themes/tabula/manifest.json">
<link rel="mask-icon" href="https://www.fietsersbond.nl/wp-content/themes/tabula/safari-pinned-tab.svg" color="#5bbad5">
```

KeePass v2.49 x64
YAFD v1.2.4.0

**Comment by navossoc** [comment]
Have you tested the [1.2.5.0-pre](https://github.com/navossoc/KeePass-Yet-Another-Favicon-Downloader/releases/tag/v1.2.5.0-pre) release?

Looks fine for me.
I'm planning to release it as the new stable.

**Comment by Eddict** [comment]
nope, but sure, will do!

**Comment by Eddict** [comment]
just tried, still Not Found. double checked the version from Tools - Plugins but that one still shows 1.2.4.0
but convinced i am using 1.2.5.o-pre (filesize 24407 bytes, compared to 16324 for 1.2.4.0). location in that Plugins popup points to correct location

**Comment by navossoc** [comment]
Yeah, the version still 1.2.4.0 because it's not fully released yet.

Weird, what OS are you using?

Have you tested the custom download provider? (if you have this option you are using the next version).
https://github.com/navossoc/KeePass-Yet-Another-Favicon-Downloader/pull/62#issuecomment-947184578

**Comment by Eddict** [comment]
Win10 21H1 (19043.1348)

yes with the custom provider (Google) it does work! so just not with the 'normal' way

**Comment by navossoc** [comment]
Well, the default option works on the principle of best effort (like internet) 😝 .
There is many things that can affect the results, that is why we added the custom provider option.

Officially released [v1.2.5.0](https://github.com/navossoc/KeePass-Yet-Another-Favicon-Downloader/releases/tag/v1.2.5.0).
